source/_docs/z-wave/control-panel.markdown

```diff
@@ -0,0 +1,106 @@
+* **is_zwave_plus** True for any Z-Wave Plus devices (note that controllers always report *false*, regardless of whether they are Plus devices or not)
+* **lastRequestRTT** The Round Trip Time of the last request
+* **lastResponseRTT** The Round Trip Time of the response to the last request
+* **manufacturer_name** The name of the manufacturer, as supplied by OpenZWave
+* **max_baud_rate** The maximum bandwidth the device supports, most modern devices will support 40,000 or higher
+* **new_entity_id** In 0.47, Home Assistant introduced a new naming convention for entities, this shows the new naming convention
+* **node_id** The unique node ID of this node
+* **node_name** The base name of this node, this is used to build the entity ID of all entities of this node
+* **old_entity_id** If `new_entity_ids: false` has been configured, then this is the entity_id that will be used. Support for this will be removed in the future
+* **product_name** The product name of the device, as supplied by OpenZWave
+* **query_stage** The query stage for this device (see [here](/docs/z-wave/query-stage/) for details)
+* **receivedCnt** The number of messages received from the device
+* **receivedDups** The number of duplicate messages received from the device
+* **receivedTS** The date and time the last message was received from the devices
+* **receivedUnsolicited** How many unsolicited messages were received
+* **retries** How many retries have been made to send messages to this node
+* **sentCnt** How many messages have been sent to the node
+* **sentFailed** How many messages that were sent weren't acknowledged
+* **sentTS** The date and time the last message was sent to the ndoe
+* **wake_up_interval** *Battery powered devices only* - the wakeup interval of the device, in seconds
+
+### {% linkable_title Node Values %}
+
+Allows you to rename the entities of the node. For example, maybe for the sensor `front_door`, you want to rename the value `seismic_intensity` to `shake`. The `entity_id` for that sensor will then change from `sensor.front_door_seismic_intensity` to `sensor.front_door_shake`.
+
+### {% linkable_title Node group associations %}
+
+Where the device supports the *Association* command class, this will allow you to associate the device with another. OpenZWave will automatically associate the device with the controller, to provide instant updates when the device doesn't support the *Hail* command class.
+
+You can use this to enable one device to directly control another. This is primarily useful for remote controls that operate lights or switches, or where you want to have multiple devices operate as one.
+
+There may be multiple groups, that are used for different purposes. The manual of your device will explain what each group is for.
+
+### {% linkable_title Node config options %}
+
+You can set the *wakeup* interval (in seconds) of the device, this is shown for all devices, but only applies to battery powered devices.
```

Comment:
This is only shown for devices reporting back the wakeup command class. Thus should only be battery powered devices.